src/components/layout/BreakingNews.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+const BreakingNews = ({ firstArticle }) => {
+  if (firstArticle) {
+    return (
+      <div>
```

Comment:
Why 2 div?

Reply:
we wanted extra padding. But it works without too, so I removed it